int32 HOME = 0
int32 MOVEVIALINITIALPOSITIONTOPUMP = 1
int32 MOVEVIALPUMPTOSTIRRER = 2
int32 MOVEVIALSTIRRERTOINITIALPOSITION = 3
int32 MOVEVIALHEATERTOSCALE = 4
int32 MOVEVIALSCALETOHEATER = 5
int32 MOVEVIALSCALETOINITIALPOSITION = 6
int32 MOVESTIRRERTOHOME = 7
int32 MOVEPUMPTOHOME = 8
int32 MOVESCALETOHOME = 9
int32 GRIPPERCLOSE = 10
int32 GRIPPEROPEN = 11

int32 panda_command
